pom.xml

```diff
@@ -6422,6 +6430,20 @@
           <uniqueVersion>true</uniqueVersion>
         </repository>
       </distributionManagement>
+      <pluginRepositories>
+        <pluginRepository>
+          <id>maven-vendor</id>
+          <url>https://packages.nuxeo.com/repository/maven-vendor</url>
+          <name>Nuxeo maven plugins repository</name>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <updatePolicy>always</updatePolicy>
+            <enabled>true</enabled>
+          </snapshots>
+        </pluginRepository>
+      </pluginRepositories>
```

Comment:
This section can also be global.

Reply:
The plugin related sections break the TnP if set as global because it has no way to know the new repositories.